openassessment/xblock/resolve_dates.py

```diff
@@ -46,7 +58,11 @@ def _parse_date(value, _):
         try:
             return parse_date(value).replace(tzinfo=pytz.utc)
```

Comment:
Just out of curiosity - should we be respecting the user's timezone here too, when parsing dates?